pages/signUp-signIn.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../style/style.css" />
+    <title>Sign Up/In Page</title>
+  </head>
+  <body class="signup-page">
+    <header></header>
+    <section class="container">
+      <div class="SwitchBtn">
+        <button id="signInBtn">SIGN IN</button>
+        <button id="signUpBtn">SIGN UP</button>
+      </div>
+      <div class="LogoDiv">
+        <img src="../images/Logo.svg" alt="" />
+      </div>
+      <div id="signUpDiv" class="InputDiv">
+        <input id="uUserName" type="text" placeholder="User Name" />
+        <input id="uEmail" type="email" placeholder="Email" />
+        <input id="uPassword" type="password" placeholder="Password" />
+        <input
+          id="uConfPassowrd"
+          type="password"
+          placeholder="Confirm Password"
+        />
+        <button>Sign up</button>
+      </div>
+      <div id="signInDiv" class="InputDiv">
+        <input id="IEmail" type="email" placeholder="Email" />
+        <input id="IPassword" type="password" placeholder="Password" />
+        <a href="#">Forgot Password</a>
+        <button>Sign In</button>
```

Comment:
Pay attention, please add here id property **"signInBtn"**  to get access to this specific button in Javascript file

Reply:
This is only for styling thank you for suggestion .

Reply:
I don't have to add any id for this buttons the way I write the CSS code